.github/workflows/main.yml

```
@@ -1,12 +1,9 @@
-name: Main collector CI
+name: Main collector driver build
 
 on:
   push:
     branches:
-      - master
-      - release-*
-    tags:
-      - 3.*.*
+      - drivers
   pull_request:
```

Do we want to limit the workflow to PRs against the drivers branch?
```suggestion
  pull_request:
    branches:
    - drivers
```